src/request/email.ts

```diff
@@ -0,0 +1,36 @@
+import crypto = require('crypto');
```

Comment:
Should we use `import * as crypto from 'crypto'` here instead? I think using the `import * as [...] from [...];` would be more consistent.

This comment applies to other uses of `import [...] = require(...);` too.

Reply:
`import * as...` works, except for 'lodash.clonedeep' for some reason...